index.html

```diff
@@ -31,12 +31,133 @@ <h1 class="about-title">I’m Martin.Glad to see you!</h1>
         </p>
         <p class="connection">LET’S CONNECT</p>
         <ul>
-            <li><a href="#"><img src="image/Vector (15).png" alt="twitter"></a></li>
+            <li ><a href="#"><img src="image/Vector (15).png" alt="twitter"></a></li>
             <li><a href="#"><img src="image/Linkedin icon.png" alt="linkedin"></a></li>
             <li><a href="#"><img src="image/Vector (11).png" alt="mail"></a></li>
             <li><a href="#"><img src="image/Vector (12).png" alt="github"></a></li>
             <li><a href="#"><img src="image/Vector (13).png" alt="peace"></a></li>
         </ul>
     </section>
+    <!-- WORK SECTION -->
+    <section class="work">
+        <!-- CARD 1 -->
+        <div class="card-1">
+            <img src="image/Snapshoot Portfolio.png" alt="image1" >
+            <h1 class="tonic">Tonic</h1>
```

Comment:
Great job on applying the correct color for the title! However, there are still some adjustments that need to be made:

- [ ] It appears that you're using the `<h1>` tag for the card titles. While this is allowed, it's generally recommended to have only one `<h1>` tag per webpage for better accessibility and clarity. Consider replacing the `<h1>` tag with a more appropriate heading tag, such as `<h2>` to `<h6>`, depending on the hierarchy and relevance of the titles within your code. This will ensure a logical heading structure and improve accessibility for users. You can refer to the [documentation on heading elements](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements#accessibility_concerns) for more information.
- [ ] The font styles for the title do not match the design specifications. Please carefully review the design and update your code to match the specified font face, size, weight, and other relevant styles for the titles.
![image](https://github.com/mar4444/re-portfolio/assets/55840999/665b9b1c-7019-4f4b-bbc1-bd246b6352a3)
- [ ] Additionally, the current margin-top between the title and the image does not match the desired gap in the design. I recommend using flexbox to create a wrapper around the title and image, and then applying the appropriate spacing using the `gap` property. This will ensure consistent and accurate spacing between the title and image.

Please address these adjustments to improve the visual appeal and accessibility of your card titles. Keep up the good work! 👏🌟